client/http_sender.go

```diff
@@ -123,22 +120,20 @@ func (s *httpSender) post(call proto.Call) error {
 
 		resp, err := s.client.Do(req)
 		if err != nil {
-			return retry.Continue, err
+			continue
```

Comment:
I think I agree in general, but in this case we'd only see an HTTP protocol error